app/src/main/java/org/simple/clinic/introvideoscreen/IntroVideoScreen.kt

```diff
@@ -18,7 +18,6 @@ import org.simple.clinic.R
 import org.simple.clinic.ReportAnalyticsEvents
```

Comment:
Can we also remove the `IntroVideoScreenKey` class as well, please?

Reply:
This has been removed in the later commit, please check

Reply:
The `IntroVideoScreenKey` still seems to be present in this branch. Please, check that.